pkg/cpi/commonUtils.go

```diff
@@ -21,6 +23,31 @@ type TokenParameters struct {
 	Client                       piperhttp.Sender
 }
 
+// CpiServiceKey contains information about a CPI service key
```

Comment:
The variable has changed and so should this documentation comment.

Reply:
I went to change this but realized I need to know first if we should change the variable to be `APIServiceKey` (below comment)